src/components/popup/details/vendors/vendors.jsx

```diff
@@ -94,35 +96,14 @@ export default class Vendors extends Component {
 				<div class={style.vendorContent}>
 					<table class={style.vendorList}>
 						<tbody>
-						{vendors.map(({ id, name, policyUrl, purposeIds, legIntPurposeIds, featureIds }, index) => (
+						{vendors.map(({ id, name, policyUrl, purposeIds=[], legIntPurposeIds=[], featureIds=[] }, index) => (
```

Comment:
Is it necessary? I thought our vendor list is already validated by our backend systems (properties like purposeIds, legIntPurposIds and featureIds are empty arrays if they does not exist)

Reply:
No, but at no cost we secure ourselves against the edge case in which these arrays don't reach our component